ui/src/main/resources/Meeting/Code/WebHomeSheet.xml

```diff
@@ -40,6 +40,26 @@
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
+#set($query = "from doc.object(Meeting.Code.MeetingClass) as meeting where doc.fullName &lt;&gt; 'Meeting.Code.MeetingTemplate'")
```

Comment:
The template must be migrated as well, so this query can be simplified.

Reply:
It would be clearer to move all this code in a separate page and call a macro from this one.